web/domain/persistence/dao.py

```
@@ -0,0 +1,261 @@
+        ceo_name:Optional[str] = None,
+        induty_code: Optional[str] = None,
+        market: Optional[str] = None,
+        homepage_url: Optional[str] = None,
+        headquarters_addr: Optional[str] = None,
+        founded_date: Optional[datetime] = None,
+        corporate_reg_no: Optional[str] = None,
+        business_reg_no: Optional[str] = None,
+        phone_number: Optional[str] = None,
+        overview: Optional[str] = None,
+        description: Optional[str] = None,
+        id: Optional[int] = None
```

Ruff (0.14.5)

Unused function argument: `id`

(ARG001)